core/src/main/scala/org/bykn/bosatsu/PackageMap.scala

```diff
@@ -52,7 +52,24 @@ object PackageMap {
   type MapF2[A, B] = MapF3[A, A, B]
   type ParsedImp = PackageMap[PackageName, Unit, Unit, (Statement, ImportMap[PackageName, Unit])]
   type Resolved = MapF2[Unit, (Statement, ImportMap[PackageName, Unit])]
-  type Inferred = PackageMap[Package.Interface, NonEmptyList[Referant[Variance]], Referant[Variance], Program[TypeEnv[Variance], TypedExpr[Declaration], Statement]]
+  type Typed[T] = PackageMap[
```

Comment:
good change. PS: this type is just wild isn't it? I get pretty baffled messing around in this code due to how deep the types go. That said, having one set of values that go through different stages enforced by the types has been pretty cool.

Reply:
heh, something I have to remind myself when confronted with it is it's all just rewriting types. It's still just the same case class